.github/workflows/update-contributors.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           git config --local user.email 'dmitry+bot@pyroscope.io'
           if ! git diff --exit-code README.md; then
               git add README.md
-              git commit -m 'docs: updates the list of contributors in README [skip ci]'
```

Comment:
Another option would be to run it monthly but keep the `skip ci` flag to avoid having an image tied to this commit. The only necessary check would be to schedule it for Wednesday(?), to allow some buffer after our release day.

Reply:
I am unsure about bringing back the `skip ci` flag. If we have a slow week (not many PRs merged), we can get in the same situation again where we can't cut a release. If there are no PRs merged between that Wednesday and Monday, the weekly release will fail and if we decide to make the official release on the same Wednesday it can break too.